sig-scheduling/annual-report-2021.md

```diff
@@ -23,45 +24,46 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
 -->
 
    - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+     - [2249 - Multi-scheduling Profiles](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/1451-multi-scheduling-profiles) - 1.22
    - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+     - [2249 - Namespace Selector for Pod Affinity](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2249-pod-affinity-namespace-selector) - 1.22
+     - [1923 - Prefer Nominated Node](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/1923-prefer-nominated-node) - 1.22   
+     - [2458 - Resource Fit Scoring Strategy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2458-node-resource-score-strategy) - 1.22
+     - [2891 - Simplified Scheduler Config](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/2891-simplified-config/kep.yaml) - 1.22
+     - [785 - Scheduler Component Config API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/785-scheduler-component-config-api) - 1.23
+     - [2926 - Job Mutable Scheduling Directives](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2926-job-mutable-scheduling-directives) - 1.23
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+     - None
    - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - None
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - https://github.com/kubernetes-sigs/kube-scheduler-simulator is short of reviewers/owners.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - Diversity
+   - Number of contributors
+   - Meetings attendance
```

Comment:
Right, in the past we had a meeting at 5:30 EST to accommodate China timezone, but for an extended period of time no one was attending from east asia and so we converged to the existing time slot which is aligned with the time slot used by both sig-node and sig-apps.

I am supportive of restarting a bi-weekly meeting friendlier to the China timezone, I am not be able to attend every time, but the sig-scheduling co-chairs/TLs could rotate moderating the meeting. @Huang-Wei @alculquicondor what do you think?

Reply:
If no agenda we can just cancel it just like what are doing now, but it does give some people in east asia a chance to talk  or discuss if necessary.  It could be a ad-hoc meeting or bi-weekly, any style shall be work.

Reply:
I'm supportive of restarting it, maybe with a reduced cadence and we can adjust depending on attendance and accumulation of topics.

Reply:
> Right, in the past we had a meeting at 5:30 EST to accommodate China timezone

We used to do sig meeting every week, in a rotated manner - 10:00 AM PST and then 5:00 PM PST.

> I am supportive of restarting a bi-weekly meeting friendlier to the China timezone

My idea is to have the current bi-weekly as-is, i.e., time is set to 10:00 AM PST. In addition to that, add an APEC-friendly **monthly** meeting (we can stack items to the agenda). I'll try to host it. WDYT?

Reply:
sounds good to me.

Reply:
opened https://github.com/kubernetes/kubernetes/issues/108288 to track adding the new meeting.